src/supervisor.c

```diff
@@ -194,7 +194,7 @@ void svSynchronize() {
 	// HELLO message!
 	kwait(50);
 	print("\r\nPowered by PROS " FW_VERSION "\r\n" FW_DISCLAIMER
-		"\r\nPROS (C)2011-2014 Purdue ACM SIGBOTS\r\n");
+		"\r\nPROS (C)2011-2017 Purdue ACM SIGBOTS\r\n");

```

Comment:
I know it's not strictly related, but while we're here and editing `supervisor.c`...